dbt/adapters/sqlserver/connections.py

```diff
@@ -347,7 +347,7 @@ def add_query(self, sql, auto_begin=True, bindings=None, abridge_sql_log=False):
 
             # pyodbc does not handle a None type binding!
             if bindings is None:
-                cursor.execute(sql)
+                cursor2 = cursor.execute(sql)
```

Comment:
Should it be called cursor2 or is this work in progress? Looked through the code and cursor2 will not be returned in the end.

Reply:
good catch. I _think_ we don't need to return the object

```suggestion
                cursor2 = cursor.execute(sql)
```